sql/core/src/test/resources/sql-tests/inputs/pgSQL/timestamp.sql

```diff
@@ -186,23 +186,23 @@ SELECT '' AS date_trunc_week, date_trunc( 'week', timestamp '2004-02-29 15:44:17
 --   FROM TIMESTAMP_TBL
 --   WHERE d1 BETWEEN timestamp '1902-01-01'
 --    AND timestamp '2038-01-01';
-
--- [SPARK-28420] Date/Time Functions: date_part
--- SELECT '' AS "54", d1 as "timestamp",
---    date_part( 'year', d1) AS year, date_part( 'month', d1) AS month,
---    date_part( 'day', d1) AS day, date_part( 'hour', d1) AS hour,
---    date_part( 'minute', d1) AS minute, date_part( 'second', d1) AS second
---    FROM TIMESTAMP_TBL WHERE d1 BETWEEN '1902-01-01' AND '2038-01-01';
-
--- SELECT '' AS "54", d1 as "timestamp",
---    date_part( 'quarter', d1) AS quarter, date_part( 'msec', d1) AS msec,
---    date_part( 'usec', d1) AS usec
---    FROM TIMESTAMP_TBL WHERE d1 BETWEEN '1902-01-01' AND '2038-01-01';
-
--- SELECT '' AS "54", d1 as "timestamp",
---    date_part( 'isoyear', d1) AS isoyear, date_part( 'week', d1) AS week,
---    date_part( 'dow', d1) AS dow
---    FROM TIMESTAMP_TBL WHERE d1 BETWEEN '1902-01-01' AND '2038-01-01';
+-- [SPARK-28767] ParseException: no viable alternative at input 'year'
+set spark.sql.parser.ansi.enabled=false;
```

Comment:
what are we doing here? This test is for timestamp but why do we test the parser？

Reply:
To use `year` as an alias name in t[he query](https://github.com/apache/spark/pull/25410/files#diff-e0d0d3afa458470e63a363b80e00e2c4R192) below, it just turns off the ansi mode temporarily;
`year` cannot be used as an alias name with ansi=true because that is a reserved keyword: https://github.com/apache/spark/pull/25410/files#r314599685

Reply:
can't we just quote it? e.g. `select 1 as 'year'`

Reply:
We can quote or set the variable. Please, take a look at the comments: https://github.com/apache/spark/pull/25410/files/af51e524d90253d26dc848d4776328c5f8359d88#r314593244 . Do you think it is better to use backquotes instead of setting the variable?

Reply:
yea, quoting looks ok to me.

Reply:
I'd like to quote it, to not distract people from the timestamp tests

Reply:
does pgsql quote it in its test?

Reply:
In pgSQL, `year` is not reserved, so we can use it as an alias name.
https://www.postgresql.org/docs/11/sql-keywords-appendix.html
Even if its reserved, we can use it though....;
```
postgres=# select 1 as year;
 year 
------
    1
(1 row)

postgres=# create table year(t int);
CREATE TABLE
postgres=# select 1 as select;
 select 
--------
      1
(1 row)

postgres=# create table select(t int);
2019-09-06 14:44:35.490 JST [6166] ERROR:  syntax error at or near "select" at character 14
2019-09-06 14:44:35.490 JST [6166] STATEMENT:  create table select(t int);
ERROR:  syntax error at or near "select"
LINE 1: create table select(t int);
```

Reply:
> I'd like to quote it, to not distract people from the timestamp tests

@dongjoon-hyun I hope you will be not so unhappy if I use backquotes again here.